ostree_upload_server/importers/base.py

```diff
@@ -83,3 +83,7 @@ def _apply_commit_to_repo(self, commit, ref):
         except:
             target_repo.abort_transaction(None)
             raise
+
+        logging.info("updating summary...")
+        update_repo_metadata(self._repo_path, self._gpg_homedir, self._sign_key)
+        logging.info("updating summary done...")
```

Comment:
I think `flatpak build-update-repo` is pretty clear on its progress. This seems excessive.